examples/tracegen/otel-collector-config.yml

```diff
@@ -0,0 +1,25 @@
+receivers:
+  otlp:
+    protocols:
+      grpc:
+
+exporters:
+  jaeger_thrift:
+    endpoint: "http://jaeger:14268/api/traces"
```

Comment:
Please use the regular jaeger exporter with gRPC. The jaeger_thrift should be used only for edge cases.

Reply:
I found that jaeger is not working well when pushing data to its gRPC port. Do you mind to file another issue for fixing this issue so that I can also remove the jaeger_thrift usage in `examples/tracing/docker-compose.yml`?

Reply:
Created https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/7242, thanks!

Reply:
Would you like to work on that first and come back to this one?

Reply:
I think the `tracegen` tool doesn't support gRPC. I feel we should file another issue to make it available.

Reply:
fixed.